lib/server.js

```diff
@@ -93,28 +92,31 @@ module.exports = async config => {
             return server.plugins.auth.generateToken(profile);
         };
 
-        // init queue
-        const res1 = await server.inject({
```

Comment:
was this making an api POST call to itself when server starts ?

Reply:
Yes, server.inject